django/conf/urls/__init__.py

```diff
@@ -17,6 +16,8 @@
 
 
 def include(arg, namespace=None, app_name=None):
```

Comment:
@aaugustin My initial idea was that third-party URL dispatchers may choose to use `django.conf.urls`, so I kept it as a separate module. I don't have a particular strong opinion on this, so if you feel that it should still be moved to `django.urls`, I'm happy to make that change.